### PR TITLE
chore: add contributors file and update copyright tagline

### PR DESCRIPTION
**Pull request by kaf-lamed-beyt**
This Pull request addresses #111

### CODE REVIEW

File: ui/js/plugins/fasagreement/fasagreement.js [hunk 1]
```diff
@@ -1,150 +1,136 @@
 //
 // FreeIPA plugin for Fedora Account System
-// Copyright (C) 2019  Christian Heimes <cheimes@redhat.com>
+// Copyright (C) 2019  FreeIPA FAS Contributors
 // See COPYING for license
 //
 
 define([
-        'freeipa/phases',
```
Comment (abompard) [inline ui/js/plugins/fasagreement/fasagreement.js:8]: Please disable your automatic code formatter and do the copyright change again in this file. Most of those changes are unrelated to this PR.

**Review by abompard — CHANGES_REQUESTED** [review]

File: ui/js/plugins/fasagreement/fasagreement.js (hunk 1 above)
Comment (kaf-lamed-beyt) [reply ui/js/plugins/fasagreement/fasagreement.js:8]: My bad! I'll do that now. 

Thank you

**Comment by kaf-lamed-beyt** [comment]
Apologies for the multiple commits! I'll close this and start afresh
